src/main/default/lwc/navigationButton/navigationButton.js

```diff
@@ -0,0 +1,76 @@
+import { LightningElement, api } from 'lwc';
+import navigate from '@salesforce/label/c.cookbook_Navigate';
+
+/**
+ * The `navigationButton` is a button to navigate through the steps of the case creation process.
+ * @example
+ * <c-navigation-button type="next" show-icon label="Next"></navigation-button>
+ */
+export default class NavigationButton extends LightningElement {
+  labels = {
+    navigate
+  };
+
+  /**
+   * The type of the navigation button.
+   * @type {'next'|'previous'}
+   * @defaultValue `'next'`
+   */
+  @api type = 'next';
+
+  /**
+   * The label to be shown in the button.
+   * @type {string}
+   * @defaultValue `'Navigate'`
```

Comment:
Since the allowed types are `next` and `previous` it seems a little odd to me to have the button text default to `Navigate`. Why not use `Next` and `Previous` as the default text?